templates/integrations.html

```diff
@@ -0,0 +1,177 @@
+        </ul>
+      </div>
+    </div>
+  </section>
+  <section class="p-section--shallow">
+    <div class="row--50-50">
+      <div class="u-fixed-width"><hr class="p-rule--muted"></div>
+      <div class="col">
+        {{ image(
+            url="https://assets.ubuntu.com/v1/a1c6a3fd-Integrations%202.png",
+            alt="",
+            height="340",
+            width="600",
+            hi_def=True,
+            loading="auto",
+          ) | safe
+        }}
+      </div>
+      <div class="col">
+        <p>When your model contains two charms you wish to connect, all you have to do is to run juju integrate from the Juju controller and specify the two charms to be connected.</p>
+      </div>
+      <div class="u-fixed-width"><hr class="p-rule--muted"></div>
+    </div>
+  </section>
+  <section class="p-section--deep">
+    <div class="row--50-50">
+      <div class="col">
+        {{ image(
+            url="https://assets.ubuntu.com/v1/39f6c093-integrations%203.png",
+            alt="",
+            height="173",
+            width="600",
+            hi_def=True,
+            loading="auto",
+          ) | safe
+        }}
+      </div>
+      <div class="col">
+        <p>Juju will automatically create an integration between these two services. You generally do not need to worry about anything else.</p>
+        <p>The integrated applications communicate directly with each other and the communication is not mediated by the juju controller.</p>
+      </div>
+    </div>
+  </section>
+  <section class="p-section--shallow">
+    <div class="row--50-50">
+      <div class="u-fixed-width"><hr class="p-rule--muted"></div>
+      <div class="col">
+        <h1 class="p-heading--2">Creating a working model</h1>
+        <div style="padding-left: 2rem; padding-top: 2rem;">
+        {{ image(
+            url="https://assets.ubuntu.com/v1/0021ecae-Working%20model.png",
+            alt="",
+            height="512",
+            width="522",
+            hi_def=True,
+            loading="auto",
+          ) | safe
+        }}
+        </div>
+      </div>
+      <div class="col">
+        <p>You can relate more and more services until you have a working model.</p>
+        <p>Once the integration is done, Juju will deal with all configurations and monitoring - you do not need to be an expert on each individual service! You can focus on judging if the model is a good representation of your idea or not.</p>
+        <p>The optimal configuration is persisted even if one of the applications is scaled, upgraded or the configuration changed. Juju and the charm code take care of it all.</p>
+      </div>
+    </div>
+  </section>
+</section>
+<section class="p-section">
+  <div class="u-fixed-width"><hr class="p-rule"></div>
+  <section class="p-section--shallow">
+    <div class="row--50-50">
+      <div class="col">
+        <h1 class="p-heading--2">Seamlessly connect applications across clouds with cross model integrations</h1>
```

Comment:
@lukasSerelis it feels a bit odd to have this as an h2, since it is still part of the description of how integrations work.
Is there a reason why we went with this design? 
The idea is that you keep adding charms and integrations until you have a working model

Reply:
I'm happy to make this into a more cohesive part of the section, was under the wrong impression that we want to highlight it as a separate point. Happy to make it look like this. 
![image](https://github.com/canonical/juju.is/assets/152963078/6518c17b-34a6-4ff8-88b1-fe36faaf178d)
@chillkang, once @massigori okays this please make this change

Reply:
I think the updated version makes more sense - I still believe the images are too large in relation to the text but let's go ahead with what we have